service/awsconfig/v5/resource/cloudformation/adapter/spec.go

```diff
@@ -108,3 +108,7 @@ type SmallCloudconfigConfig struct {
 type ELBClient interface {
 	DescribeLoadBalancers(*elb.DescribeLoadBalancersInput) (*elb.DescribeLoadBalancersOutput, error)
 }
+
+type Interface interface {
+	Adapt(config Config) error
+}
```

Comment:
This is what the `hydrater` is about so far. I think there should be an interface which adapters implement.

Reply:
Could you please elaborate? We have already the function type, I don't understand why this is needed.

Reply:
I added the output adapter mutation back to the loop. I removed this since it is not used yet.